EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -37,23 +37,41 @@ struct MediaPlayerView: View {
     }
 
     @ViewBuilder private func renderActionButtons() -> some View {
-        HStack {
-            renderRewind()
-                .frame(maxWidth: .infinity, alignment: .leading)
-            if viewModel.playerState == .finished {
-                renderReplay()
-            } else {
-                renderPlayPause()
+        VStack(spacing: Constants.buttonsSpacing) {
+            HStack {
+                renderLoop()
+            }
+            HStack {
+                renderRewind()
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                if viewModel.playerState == .finished {
+                    renderReplay()
+                } else {
+                    renderPlayPause()
+                }
+                renderForward(disabled: (viewModel.playerState == .finished))
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            HStack {
             }
```

Comment:
Remove empty `HStack`

Reply:
This is actually to keep the buttons aligned, future implementations will place items inside